common/templates/recordAction-ent.html

```diff
@@ -0,0 +1,20 @@
+<div ng-class="{row:rowStyle}">
```

Comment:
So rather than updating `rowStyle` in the controller, change the conditional in the HTML to check what mode the table display is in.